Content.Server/Speech/Components/RandomBarkComponent.cs

```diff
@@ -0,0 +1,63 @@
+namespace Content.Server.Speech.Components;
+
+/// <summary>
+///     Sends a random message from a list with a provided min/max time.
+/// </summary>
+[RegisterComponent]
+public sealed partial class RandomBarkComponent : Component
+{
+    /// <summary>
+    ///     Should the message be sent to the chat log?
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public bool ChatLog = false;
+
+    /// <summary>
+    ///     Minimum time an animal will go without speaking
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public int MinTime = 45;
+
+    /// <summary>
+    ///     Maximum time an animal will go without speaking
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public int MaxTime = 350;
+
+    /// <summary>
+    ///     Accumulator for counting time since the last bark
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public float BarkAccumulator = 8f;
+
+    /// <summary>
+    ///     Multiplier applied to the random time. Good for changing the frequency without having to specify exact values
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public float BarkMultiplier = 1f;
```

Comment:
Just to clarify; what does this do? Not really getting from the description

Reply:
![image](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/11f3ee90-d427-4945-b867-c6655fba439c)
It accelerates the rate at which barks accumulate. The higher the multiplier is, the more often the barks happen. Make it lower to make them bark less often.